tools/linker-process/index.bzl

```diff
@@ -0,0 +1,92 @@
+load("@build_bazel_rules_nodejs//:providers.bzl", "JSModuleInfo", "JSNamedModuleInfo")
+
+"""Gets the Bazel manifest path for a given file."""
```

Comment:
Would it make sense to elaborate somewhere in this file what a manifest is?

Reply:
I've added a little example of how these paths look like and where they usually reside in Bazel.